packages/suite-web/package.json

```diff
@@ -40,13 +40,11 @@
         "@trezor/transport": "workspace:*",
         "@trezor/trezor-user-env-link": "workspace:*",
         "@trezor/utils": "workspace:*",
-        "@types/chrome-remote-interface": "^0.31.14",
         "@types/cypress-image-snapshot": "^3.1.8",
         "@types/react": "18.2.79",
         "@types/react-dom": "18.2.19",
         "@types/react-router": "^5.1.20",
         "@types/react-router-dom": "^5.1.7",
-        "chrome-remote-interface": "^0.33.0",
```

Comment:
this used to be used in cypress tests. not sure if it is still active, haven't been maintaining it for a while.

Reply:
it was commented out so i guess it is ready to be removed

Reply:
I guess so, cc @andrcj